rskj-core/src/main/java/co/rsk/peg/Bridge.java

```diff
@@ -564,6 +573,34 @@ public Integer getBtcBlockchainInitialBlockHeight(Object[] args)
         return blockHash.getBytes();
     }
 
+    public long getBtcTransactionConfirmationsGetCost(Object[] args) {
+        Sha256Hash btcBlockHash = Sha256Hash.wrap((byte[]) args[1]);
```

Comment:
Could it be possible to get an NPE here?

Reply:
good catch